projects/ngx-d3timeline/src/lib/axis/axis.service.ts

```diff
@@ -28,7 +33,12 @@ export class AxisService {
   timeAxis$ = this.scalesService.scales$.pipe(
     map(scales =>
       this.createAxis(
-        new TimeAxisTickRenderer(scales.scaleTime),
+        new TimeAxisTickRenderer(
```

Comment:
This can be injected into the constructor, see comment below on removing other constructor parameters

Reply:
As above will address in #107